app/views/ncr/proposals/new.html.erb

```diff
@@ -19,9 +19,22 @@
       <% end %>
     </div>
     <div class="form-group">
+      <%= f.label :RWA_number %>
+      <%= f.text_field :RWA_number, disabled:true, class: 'form-control' %>
+    </div>
+    <div class="form-group">
+      <%= f.label :building_number %>
+      <%= f.select(:building_number, [], {}, {class: 'form-control', multiple: false}) do %>
+        <%= content_tag(:option, '', value: '') %>
+        <% Ncr::ProposalForm::BUILDING_NUMBERS.each do |building| %>
+          <%= content_tag(:option, building, value: building) %>
```

Comment:
Wondering if we should store the building code instead of the full text value... though there will be things like "Entire WH Complex" that don't have one. Hmm.